qa/cephfs/overrides/ignorelist_health.yaml

```diff
@@ -12,6 +12,7 @@ overrides:
       - MDS_UP_LESS_THAN_MAX
       - filesystem is online with fewer MDS than max_mds
       - POOL_APP_NOT_ENABLED
+      - do not have an application enabled
```

Comment:
Looks fine. I ran into the same with INLINE_DATA and its human friendly equivalent. Have things changed w.r.t. which format type is clog'ed?

Reply:
It's just more fallout from the clog changes I think.